.circleci/config.yml

```diff
@@ -0,0 +1,61 @@
+# These environment variables must be set in CircleCI UI
+#
+# DOCKERHUB_REPO - docker hub repo, format: <username>/<repo>
+# DOCKER_USER
+# DOCKER_PASS
+#
+
+version: 2
+jobs:
+  build:
+    machine:
+      enable: true
+    working_directory: ~/mozilla/telemetry-analysis-service
+    steps:
+      - checkout
+      - run: ./bin/build
+
+  test:
+    machine:
+      enable: true
+    working_directory: ~/mozilla/telemetry-analysis-service
+    steps:
+      - checkout
+      - run: sudo apt-get update
+      - run: sudo apt-get install python-dev
+      - run: sudo pip install --upgrade pip
+      - run: pip install docker-compose
+      - run: docker info
+      - run: docker --version
+      - run: ./bin/test
+
+  deploy:
+    machine:
+      enable: true
+    working_directory: ~/mozilla/telemetry-analysis-service
+    steps:
+      - checkout
+      - run: ./bin/build && ./bin/deploy
```

Comment:
Any specific reason why this is called in one line instead of in two separate ones?

Reply:
I read somewhere that runs run in separate subshells

Reply:
Ah, in parallel?

Reply:
ah no. I think similar to how Dockerfiles construct images in layers, separate but sequential.